autoload/neomake.vim

```diff
@@ -236,9 +243,9 @@ function! s:AddExprCallback(maker) abort
     let place_signs = get(g:, 'neomake_place_signs', 1)
     let list = file_mode ? getloclist(a:maker.winnr) : getqflist()
 
-    while s:neomake_list_nr < len(list)
-        let entry = list[s:neomake_list_nr]
-        let s:neomake_list_nr += 1
+    while b:neomake_list_nr < len(list)
```

Comment:
Good point, this probably applies to the other place where I used `b:`.
There is another reason this does not work with `Neomake!`: entries in the quickfix list are going to be processed several times.